src/arch/svm/SpokeUtils.ts

```diff
@@ -89,22 +90,63 @@ type ProtoFill = Omit<RelayData, "recipient" | "outputToken"> & {
   outputToken: SvmAddress;
 };
 
+export function getSlot(
+  provider: SVMProvider,
+  commitment: Commitment,
+  logger: winston.Logger,
+  maxRetries = 2
+): Promise<bigint> {
+  return _callGetSlotWithRetry(provider, commitment, 0, maxRetries, logger);
+}
+
+async function _callGetSlotWithRetry(
+  provider: SVMProvider,
+  commitment: Commitment,
+  retryAttempt: number,
+  maxRetries: number,
+  logger: winston.Logger
+): Promise<bigint> {
+  try {
+    return await provider.getSlot({ commitment }).send();
+  } catch (err) {
+    if (!isSolanaError(err)) {
+      throw err;
+    }
+
+    const { __code: code } = err.context;
+
+    logger.debug({
+      at: "getSlot",
+      message: "Caught error from getSlot()",
+      errorCode: code,
+      commitment,
+      retryAttempt,
+      maxRetries,
+    });
```

Comment:
Minor simplification - we can reduce the number of places where we `throw` by changing the if() statement.

```suggestion
    if (isSolanaError(err)) {
      const { __code: code } = err.context;
      logger.debug({
        at: "_getSlotWithRetry",
        message: "Caught error from getSlot()",
        code,
        commitment,
        retryAttempt,
        maxRetries,
      });
    }
```